datasets/_global/tokyo_mou/crawler.py

```diff
@@ -0,0 +1,141 @@
+import httpx
+import re
+
+from lxml import etree
+from lxml import html
+from pprint import pprint
+from time import sleep
+
+from zavod import Context
+
+imo = "8817007"
+MAX_RETRIES = 5
+RETRY_DELAY = 5  # seconds
+
+
+def solve_arithmetic(expression: str) -> str:
+    match = re.search(r"(\d+)\s*([+\-*/])\s*(\d+)", expression)
+    if not match:
+        raise ValueError(f"Invalid CAPTCHA expression: {expression}")
+    a, op, b = match.groups()
+    return str(eval(f"{a}{op}{b}"))
+
+
+def crawl(context: Context):
+    with httpx.Client(http2=False, follow_redirects=True) as client:
```

Comment:
any reason why you need to use httpx rather than the usual context.fetch_* or if they don't work, the requests session via `context.http.*`?